src/commands/dev/audit/messages.ts

```diff
@@ -75,14 +73,12 @@ const messages = Messages.load('@salesforce/plugin-dev', 'audit.messages', [
 
 export default class AuditMessages extends SfCommand<AuditResults> {
   public static readonly summary = messages.getMessage('summary');
-  public static readonly description = messages.getMessage('description');
   public static readonly examples = messages.getMessages('examples');
 
   public static readonly flags = {
     'project-dir': Flags.directory({
       summary: messages.getMessage('flags.project-dir.summary'),
       char: 'p',
-      description: messages.getMessage('flags.project-dir.description'),
```

Comment:
You deleted this description ^ But there are two other flag descriptions below that could be removed from this file and from the corresponding messages file.